### PR TITLE
polish CSS template code (addresses CSS only for issue #104)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: Assets/WebGLTemplates/WebVR/webvr.css [hunk 1]
```diff
@@ -1,16 +1,16 @@
 html, body {
   overflow: hidden;
   font-size: 16px;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  font-family: BlinkMacSystemFont, -apple-system, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```
Comment (cvan) [inline Assets/WebGLTemplates/WebVR/webvr.css]: FYI: this is for earlier versions of Edge and IE 10 (which will choke on this whole list of font-families and render the text in plain serif if it encounters `-apple-system` as first font-family)

More info in [this article](https://booking.design/implementing-system-fonts-on-booking-com-a-lesson-learned-bdc984df627f):
> don’t use `-apple-system` at the head of a shorthand font declaration

I can add a comment if it's useful. Just thought I'd explain the rationale here for review purposes.

Comment (delapuente) [reply Assets/WebGLTemplates/WebVR/webvr.css]: Not blocking but please, add it.

Comment (cvan) [reply Assets/WebGLTemplates/WebVR/webvr.css]: done, thanks!

**Comment by cvan** [comment]
thanks, @delapuente!

**Pull request merged**